File: platforms/kotlin/examples/scene/model/translate.kt
import org.fragmentcolor.*

val renderer = Renderer()
val mesh = Mesh()
mesh.addVertex( Vertex(listOf(0.0f, 0.0f, 0.0f)).set("normal", floatArrayOf(0.0f, 1.0f, 0.0f)).set("uv0", listOf(0.0f, 0.0f)), )

val model = Model(mesh, Material.pbr())
model.translate(listOf(5.0f, 0.0f, -2.0f))

File: platforms/swift/examples/scene/model/translate.swift
import FragmentColor

let renderer = Renderer()
let mesh = Mesh()
try mesh.addVertex(
    try Vertex([0.0, 0.0, 0.0]).set(Vertex.nORMAL, [0.0, 1.0, 0.0]).set(Vertex.uV0, [0.0, 0.0]),
)

let model = Model(mesh, Material.pbr()?)
model.translate([5.0, 0.0, -2.0])